src/pages/components/button-group.mdx

```diff
@@ -10,13 +10,20 @@ import ComponentFeedback from 'components/ComponentFeedback';
 <ComponentDescription name="Button group" type="ui" />
 
 <AnchorLinks>
-  
+
+<AnchorLink>Content guidance</AnchorLink>
 <AnchorLink>Design and functional specifications</AnchorLink>
 <AnchorLink>Development documentation</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Notes |
+| -------- | --------- | ---------- | ---------- | ----------| 
+| [CTA (button)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | Yes | Two or more buttons | There is one primary button shown at the end. The rest of the buttons are secondary.  |
```

Comment:
I think we have a circular dependency here. 

> Button Group > CTA (button) > Button Group and so on and so on.

The dependency here is actually Carbon's Button component.

https://www.carbondesignsystem.com/components/button/usage/

Here is the correct dependency tree.
> CTA > Button group > Button (Carbon)

Reply:
Wow, didn't see that one coming. 
So I don't think we would necessarily want to just send the user to the Carbon button component cause that takes them out of the site. So what if we post the content model here and make a note at the bottom with a link to the Carbon site? @photodow

Reply:
Just talked with Jenny and we are deciding to link to the Carbon button component and I'm just going to make sure to use target="_blank".